pay-api/src/pay_api/services/eft_statements.py

```diff
@@ -0,0 +1,126 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Service to support EFT statements."""
+from __future__ import annotations
+
+from sqlalchemy import and_, func, select
+
+from pay_api.models import EFTCreditInvoiceLink as EFTCreditInvoiceLinkModel
+from pay_api.models import Invoice as InvoiceModel
+from pay_api.models import PaymentAccount as PaymentAccountModel
+from pay_api.models import Statement as StatementModel
+from pay_api.models import StatementInvoices as StatementInvoicesModel
+from pay_api.models import db
+from pay_api.utils.enums import EFTCreditInvoiceStatus, InvoiceStatus, PaymentMethod
+
+
+class EFTStatements:
```

Comment:
Splitting this out as we are getting more complex queries for returning statement break down data specifically for paying an EFT statement for staff screens.